python/ray/experimental/workflow/__init__.py

```diff
@@ -1,4 +1,6 @@
-from ray.experimental.workflow.api import step, run, resume, get_output
+from ray.experimental.workflow.api import (step, actor, run, resume,
+                                           get_output, get_actor)
 from ray.experimental.workflow.workflow_access import WorkflowExecutionError
 
-__all__ = ("step", "run", "resume", "get_output", "WorkflowExecutionError")
+__all__ = ("step", "actor", "run", "resume", "get_output", "get_actor",
```

Comment:
@suquark  
For the actor, I'm thinking about whether we'd like to sync with ray?  Ray use ray.remote for both task and actors, should we use step for both task and actors in workflow?

Reply:
I think the semantics are quite different (remote + actor = remote actor; but step + actor != virtual actor). To unify the semantics, I think something like `@workflow.primitive` would be good for both step/actor (I like `primitives` better because our step is not a singleton, it can generate many steps by keep calling `.step()`).

Reply:
Let's keep it as it is right now. I'll write a doc to collect feedback from people about the API.